scalafmt-tests/src/test/resources/scala3/TypeLambda.stat

```diff
@@ -12,6 +12,29 @@ maxColumn = 30
 ===
 type   Tuple   =  [  TheOnlyTypeInTheTuple ] =>> (TheOnlyTypeInTheTuple, TheOnlyTypeInTheTuple)
 >>>
+type Tuple =
+  [TheOnlyTypeInTheTuple] =>> (
+      TheOnlyTypeInTheTuple,
+      TheOnlyTypeInTheTuple
+  )
+<<< type lambda long, align open
+maxColumn = 30
+align.openParenCallSite = true
+===
+type   Tuple   =  [  TheOnlyTypeInTheTuple ] =>> (TheOnlyTypeInTheTuple, TheOnlyTypeInTheTuple)
+>>>
+type Tuple =
```

Comment:
The result here seems to be identical to the next test and the previous one. What is this actually testing?

Shouldn't the last one be:
```scala
type Tuple = 
  [TheOnlyTypeInTheTuple] = (
                            TheOnlyTypeInTheTuple,      
                            TheOnlyTypeInTheTuple  
                            )
```

Reply:
yep... let me check, perhaps maxColumn should be larger.

Reply:
found it. it was a test for the next batch of commits which i haven't submitted yet :) will move it there.

Reply:
are we good to go?